examples/mocks/src/retry-dynamic-import.ts

```diff
@@ -0,0 +1,17 @@
+export async function retryDynamicImport() {
+  let retryTimes = 0
+  const load = async () => {
+    try { // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-expect-error
```

Comment:
Why do you need this here?

Reply:
This case is for dynamic import in browser, bundled by webpack.  
The modules dynamically imported are from remote over network, so it maybe failed cause network exception or any unknown reason.  
We want to retry import several times for reducing the final failed possibility.

Reply:
Emmm , did you mean the comments upon?  
Without the comments, the ts language service will emit an error diagnostic message as below 
![image](https://github.com/vitest-dev/vitest/assets/16648966/60761e53-102e-4f5e-b195-c4311b7ec677)

Reply:
I have appended a commit, changed the import path to have extension, to avoid this ts check issue.